src/llmcompressor/utils/helpers.py

```diff
@@ -75,6 +75,7 @@
 RECIPE_METADATA_KEY = "__metadata__"
 FRAMEWORK_METADATA_KEY = "framework_metadata"
 ROOT_PATH = Path(__file__).resolve().parents[1]
+_sentinel = object()
```

Comment:
do we need both this and `_sentinel = object()` on L1070?

Reply:
looks like you have since removed

Reply:
Yeah thanks for the catch :)